app/src/main/java/org/jellyfin/androidtv/constant/QualityProfiles.kt

```diff
@@ -0,0 +1,31 @@
+package org.jellyfin.androidtv.constant
+
+enum class QualityProfiles(val quality: String) {
```

Comment:
I feel like this shouldn't be an enum, why not just a setOf() with all values?

Reply:
Since the setter for the MaxBitrate property of the UserPreferences class only seems to be accessible via the "bind" method in the OptionsItemMutable abstract class, I needed to write a different method for setting the MaxBitrate in VideoQualityController. 

**Both methods need access to the same set of values when getting/setting the MaxBitrate.**

Excuse my inexperience with Kotlin, but I'm not sure if setOf() is appropriate in that situation?

I could see something similar to OptionsItemMutable being written for this new use-case: settings like MaxBitrate and Audio Transcoding options set both in the Playback UI _and_ the Playback Preferences Screen, but that's a bit beyond my experience or available time commitment at the moment. Deciphering the playback code has been difficult.

Reply:
Copying the code in the current PlaybackPreferencesScreen to it's own file and assigning it to a variable should work for this purpose, like so:

```kotlin
val qualityOptions = setOf(
	0, // Disable
	3, 5, 7, // 10<
	10, 20, 30, 60, // 100<
	120, 300
).associate {
	val value = if (it == 0) getString(R.string.lbl_none)
	else TimeUtils.formatSeconds(context, it)
		it.toString() to value
}
```

And then just say `entries = qualityOptions` in the playbackpreferencesscreen and something similar in the video quality controller

Reply:
Hey - I see you requested a review for this. I'm on vacation for a few more days.

Reply:
Done.